composer.json

```diff
@@ -17,7 +17,7 @@
         "infection/infection": "0.26.16",
         "ocramius/package-versions": "^2.7.0",
         "sanmai/later": "^0.1.2",
-        "vimeo/psalm": "^4.30.0"
+        "vimeo/psalm": "^4.30.0 || ^5.0.0"
```

Comment:
Note: to be restricted in a future version, but for now allows better dependency upgrades upstream, and doesn't cause any trouble for us here.